### ISSUE TITLE
dist/vue-at-textarea is missing

### ISSUE THREAD

**Issue by sentiasa** [fritx/vue-at#28]

**Comment by fritx** [comment]
@sentiasa hey, yes, apology for the missing documentation:

The textarea feature came with vue-at@2.x, and it was not included in 1.x branch..

**Comment by sentiasa** [comment]
@fritx Oh that's bad not to be able to use it :(

I see you've added enhancement tag. Do you think it's early in the roadmap or should I try to find other solutions?

**Comment by fritx** [comment]
Oh I think it's not hard to bring textarea into 1.x branch.
I'm gonna to do it in couple of days once I'm free.

**Comment by fritx** [comment]
@sentiasa hey, sorry for the late update, you can try install `vue1-at` insread.

Just merged with 2.x branch and publish as a new npm entry due to some breaking changes,
rather than the legacy `vue-at@1.x`.

**Comment by sentiasa** [comment]
@fritx Great, thanks! I made it work!

I have couple of questions regarding this:
- Is it possible to customize the list rows? For example, adding a photo of user
- Is it possible to specify the selected user's id and maybe trigger a function accordingly? For being more precise when/who got selected.

Thanks again.

**Comment by fritx** [comment]
@sentiasa nice.

For question 1, as @youyi1314 said, you find more in docs:
https://fritx.github.io/vue-at/#/en/customtemplates

For question 2, might have not been supported yet, you have to parse the text yourself.
Would be good to support your request.
Anything I'm missing?

**Comment by sentiasa** [comment]
@fritx yes, that answers all. For the question 2, I handled it manually but that'd be nice to have this ability.

I have more small question though. Is it possible to make the mention box below rather than above? Thanks again!
